app/forms/declaration/destinationCountries/DestinationCountries.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration.destinationCountries
+
+import play.api.data.Form
+
+object DestinationCountries {
+  val formId = "DestinationCountries"
+
+  val limit = 99
```

Comment:
This `val` is related to standard journey only. Better move it to corresponding object, not the general one.
And the name is too general. Maybe `coutriesOfRoutingMaxAmount` ?

Reply:
I'll stay with this object because it will be difficult to use standardForm and supplementaryForm or formId easy inside the controller